From 89a7a63b9ad06aebdcb16f516a21dea802034004 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 23 Sep 2021 08:32:31 +0200
Subject: [PATCH 281/529] ARM: configs: disable framebuffer console on
 fragment-01-multiv7_cleanup

Disable the framebuffer console to protect the U-boot splash
screen displayed on the panel during the kernel boot.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Change-Id: I7a9cb4e59e002414573e2160a15f76b8fbf29476
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228333
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280092
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -52,6 +52,7 @@
 # CONFIG_ENABLE_MUST_CHECK is not set
 # CONFIG_ENABLE_WARN_DEPRECATED is not set
 # CONFIG_EXTCON is not set
+# CONFIG_FRAMEBUFFER_CONSOLE is not set
 # CONFIG_GCC_PLUGINS is not set
 # CONFIG_GPIO_PCA953X is not set
 # CONFIG_GPIO_PCF857X is not set
